Single-Sign-on/index.md

```diff
@@ -0,0 +1,102 @@
+---
+description: Single Sign-on
+keywords: Single Sign-on, SSO, sign-on
+title: Docker Single Sign-on
+---
+
+Docker Single Sign-on (SSO) allows users to authenticate using their identity providers (IdPs) to access Docker. Docker currently supports SAML 2.0 and Azure AD IdPs through Auth0. You can enable SSO on organization's that are part of the Docker Business subscription. To upgrade your existing account to a Docker Business subscription, see [Upgrade your subscription](../subscription/upgrade/){:target="blank" rel="noopener" class=""}.
+
+When SSO is enabled, users are redirected to your provider’s authentication page to authenticate using SSO. They cannot authenticate using their personal login credentials (Docker ID and password).
+
+Before enabling SSO in Docker Hub, administrators must work with their identity provider to configure their IdP to work with Docker Hub. Docker provides the Assertion Consumer Service (ACS) URL and the Entity ID. Administrators use this information to establish a connection between their IdP server and Docker Hub.
+
+After establishing the connection between the IdP server and Docker Hub, administrators log into the organization in Docker Hub and complete the SSO enablement process. See the section Enable SSO in Docker Hub for detailed instructions.
+
+To enable SSO in Docker Hub, you need the following:
+
+* **SAML 2.0**: Entity ID, ACS URL, Single Logout URL and Certificate Download URL
+* **Azure AD**: Client ID (a unique identifier for your registered AD application), Client Secret (a string used to gain access to your registered Azure AD application), and AD Domain details
+
+We currently support enabling SSO on a single organization. If you have any users in your organization with a different domain (including social domains), they will be added to the organization as guests.
+
+## SSO prerequisites
+
+* You must first notify your company about the new SSO login procedures
```

Comment:
We can insert the note about users may wish to maintain separate account for personal and company use?